<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TrackAreaWidget" name="trackareawidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="pianorollDock">
   <property name="windowTitle">
    <string>Pianoroll-panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="PianoRollWidget" name="pianoroll" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="channelsDock">
   <property name="windowTitle">
    <string>Channels-pannel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="channelsDockWidget">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="ChannelStripContainer" name="channelsContainer" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionChannels_pannel"/>
   <addaction name="actionPianoroll_panel"/>
   <addaction name="actionInserteffects_panel"/>
  </widget>
  <widget class="QDockWidget" name="headerDock">
   <property name="features">
    <set>QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="headerDockWidget">
    <layout class="QHBoxLayout" name="headerDockWidgetLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="player" native="true">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnStop">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPause">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>play</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>304</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
    <zorder>horizontalSpacer</zorder>
    <zorder>widget</zorder>
    <zorder>player</zorder>
   </widget>
  </widget>
  <action name="actionChannels_pannel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Channels pannel</string>
   </property>
   <property name="toolTip">
    <string>Channels pannel</string>
   </property>
  </action>
  <action name="actionPianoroll_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pianoroll panel</string>
   </property>
  </action>
  <action name="actionInserteffects_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Inserteffects panel</string>
   </property>
   <property name="toolTip">
    <string>Inserteffects panel</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PianoRollWidget</class>
   <extends>QWidget</extends>
   <header>pianoroll/pianorollwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrackAreaWidget</class>
   <extends>QWidget</extends>
   <header>trackarea/trackareawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelStripContainer</class>
   <extends>QWidget</extends>
   <header>channel/channelstripcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
